src/radical/pilot/configs/resource_tacc.json

```diff
@@ -1,5 +1,42 @@
 
 {
+    "frontera_interactive": {
+        "description"                 : "Petascale computing system at the Texas Advanced Computing Center (TACC)",
+        "notes"                       : "Specify this resource in the pilot description and run RP within a compute node(s)",
+        "schemas"                     : ["local", "ssh"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "ssh://localhost/",
+            "filesystem_endpoint"     : "sftp://localhost/"
+        },
+        "local"                       :
+        {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+        "resource_manager"            : "SLURM",
+        "cores_per_node"              : 56,
+        "gpus_per_node"               : 0,
+        "agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "launch_methods"              : {
+                                         "order": ["SRUN"],
+                                         "SRUN" : {
+                                             "pre_exec_cached": []
```

Comment:
do we need to add corresponding modules load here? (as for `frontera`)

Reply:
loading modules like the one we have in the `non-interactive` resource config ones lead to an error in MPI binding, as it overrides the preloaded module by the interactive job.

Reply:
I would think yes and no - (1) I see that user could be completely responsible which modules should be loaded, since it is an interactive mode and user is able to control the environment; (2) RP is responsible to set a correct environment (load modules) to run a corresponding LM. I would follow the 2nd case, and extra modules, which are required for user, should go into task's `pre_exec`

Let's also see what's @andre-merzky 's opinion on this

Reply:
Good points, thanks Mikhail, happy to accommodate any.

Reply:
I don't even understand the problem :-P  Why would the environment of the client interact with the environment of the LM?  Or, in other words, why does it matter if the client lives on a login node or a compute node?  After the bootstrapper run, the agent env should be basically identical (assuming the bootstrapper loads the same modules)...